conda_package_handling/api.py

```diff
@@ -80,6 +82,38 @@ def _convert(fn):
             _convert(fn)
 
 
+def gpg_sign(fns, out_folder=None, tool='gpg', identity=None, homedir=None, **kw):
```

Comment:
Given that this has a tool parameter do you expect to be able to gpg sign with other tools? What other tools will work?

Reply:
Would openssl make sense?  I have run across it in trying to learn about HSM stuff.  Since we already have openssl as a dep (via python) maybe that's a nicer way to do things?

https://eclipsesource.com/blogs/2016/09/07/tutorial-code-signing-and-verification-with-openssl/

Reply:
@soapy1 also pointed me at this helpful document: https://www.dnssec.cz/files/nic/doc/hsm.pdf

Reply:
It might be useful to have a plug-able type interface here? Like from what I understand about HSMs they support different interfaces depending on the brand. And then, internally we maybe want to be able to use that. But maybe other people might want to gpg sign packages? This stack exchange has some interesting thoughts https://security.stackexchange.com/questions/4881/use-rsa-keypair-for-pgp-encryption-and-decryption

Reply:
> Given that this has a tool parameter do you expect to be able to gpg sign with other tools? What other tools will work?

Different distros have different binary names for gpg. Some call it just 'gpg' and some call it 'gpg2'. Some have both, some have only one.